R/compareScenConf.R

```diff
@@ -0,0 +1,110 @@
+#' take two REMIND scenario-config*.csv files and print the difference,
+#' comparing it to a default.cfg
+#'
+#' @param fileList vector containing two csv file paths. If empty, user can select
+#' @param configfile path to configfile. If empty, uses './default.cfg'
+#' @param renamedColumns vector with old and new column names such as c("old1" = "new1", "old2" = "new2"))
+#' @param renamedRows vector with old and new row names such as c("old1" = "new1", "old2" = "new2"))
+#' @author Oliver Richters
+#' @examples
+#'
+#'  \dontrun{
+#'     compareScenConf(fileList = c("scenario_config1.csv", "scenario_config2.csv"),
+#'     configfile = "default.cfg",
+#'     renamedColumns = c("old1" = "new1", "old2" = "new2"),
+#'     renamedRows = c("old3" = "new3", "old3" = "new4"))
+#'   }
+#'
+#' @return nothing. Prints a lot of messages.
+#' @importFrom utils read.csv2
+#' @export
+compareScenConf <- function(fileList = "", configfile = "default.cfg", renamedColumns = NULL, renamedRows = NULL) {
+  if (!file.exists(configfile)) {
+    if (file.exists(paste0("config/", configfile))) {
+      configfile <- paste0("config/", configfile)
+    } else {
+      stop(paste0("No 'default.cfg' in working directory (", getwd(), ") or unknown 'configfile' in function call."))
```

Comment:
An error message `"Config file '", configfile, "' not found in current directory or in the 'config/' subdirectory."` or something along those lines would be more useful, I think.

Reply:
Or better yet, mention `getwd()` and `configfile` in the error message.

Reply:
You are right, I changed it such that
```
compareScenConf(configfile = "quak")
```
now returns:
```
Error in compareScenConf(configfile = "quak") : 
  No configfile quak in working directory (/p/tmp/oliverr).
```